treeherder/perf/models.py

```diff
@@ -145,6 +162,25 @@ class PerformanceAlert(models.Model):
     t_value = models.FloatField(
         help_text="t value out of analysis indicating confidence "
         "that change is 'real'")
+    bug_number = models.PositiveIntegerField(null=True)
+
+    def save(self, *args, **kwargs):
+        # just a bit of extra business logic to make sure
+        # the status corresponds with other metadata
+        if self.revised_summary is not None:
+            self.status = PerformanceAlert.DUPLICATE
+        elif self.status == PerformanceAlert.DUPLICATE:
+            self.status == PerformanceAlert.UNTRIAGED
+
+        if self.bug_number is not None:
+            if self.status == PerformanceAlert.UNTRIAGED:
+                self.status = PerformanceAlert.INVESTIGATING
+        elif self.status in (PerformanceAlert.WONTFIX,
+                             PerformanceAlert.RESOLVED,
+                             PerformanceAlert.INVESTIGATING):
+            self.status = PerformanceAlert.UNTRIAGED
```

Comment:
It's the same as above. If we don't have a bug number, we can't say it's wonfix/resolved/investigating, so reset the status to untriaged.